src/midiout.c

```diff
@@ -0,0 +1,400 @@
+
```

Comment:
Whose code is this?

Reply:
This is mine, but I was looking at RtMidi and PortMidi when I implemented ALSA. This code is trivial API calls, nothing special. The ALSA API is over-complicated, though.